### PR TITLE
Remove SW Topics Suggestion form

### PR DESCRIPTION
**Pull request by KyraAssaad**
Since it doesn't work anymore

### CODE REVIEW

**Review by Vinnl — APPROVED** [review]
Did the backlog of topic ideas (and talk proposals) get transferred? (Asking because I proposed a talk a long time ago as well :P )

File: pages/solid-events.md [hunk 1]
```diff
@@ -18,8 +18,6 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 
 - Interested in speaking at an upcoming Solid World? You can fill out the [Solid World Speaker form](https://es1cz4pb7oi.typeform.com/to/nietD34f).
 
-- Have a topic suggestion for an upcoming Solid World? You can fill out the [Solid World Topic Suggestions form](https://es1cz4pb7oi.typeform.com/to/yQNMvK0N).
```
Comment (theRealImy) [inline pages/solid-events.md:21]: The new link is: https://es1cz4pb7oi.typeform.com/to/nietD34f?utm_source=SolidWorld
Sorry I cannot make a direct suggestion on deleted stuff... it seems like.

**Review by theRealImy — CHANGES_REQUESTED** [review]

**Comment by theRealImy** [comment]
> Did the backlog of topic ideas (and talk proposals) get transferred? (Asking because I proposed a talk a long time ago as well :P )

@Vinnl sorry about that. I will look into it.

File: pages/solid-events.md (hunk 1 above)
Comment (KyraAssaad) [reply pages/solid-events.md:21]: We have that link in the point above it, for being a speaker

**Review by theRealImy — APPROVED** [review]

**Pull request merged**